### PR TITLE
Move OCR code to separate class

### PR DESCRIPTION
**Pull request by sarav**
Initial set of commits I'd like to merge. I'll proceed to clean this up further.

### CODE REVIEW

**Comment by sarav** [comment]
I tested this on a few pokemon. Works fine.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
     <string name="pokemon029">Nidoran♀</string>
     <string name="pokemon032">Nidoran♂</string>
```
Comment (Blaisorblade) [inline app/src/main/res/values/strings.xml]: Can we also remove `pokemon029` and `pokemon032` like `pokemon133 `, since they're also redundant?

**Comment by Blaisorblade** [comment]
LGTM!

**Comment by Blaisorblade** [comment]
Thanks @nahojjjen for merging!

BTW, as a general rule, as I wrote on Discord: I think we should try to merge refactorings fast, in general, so they don't have to be updated too much.
Otherwise doing refactorings will get too painful.

In case of conflicts with long running branches... we'll see in each case.

File: app/src/main/java/com/kamron/pogoiv/MainActivity.java [hunk 2]
```diff
@@ -253,6 +250,7 @@ public void onClick(View v) {
 
 
         displayMetrics = this.getResources().getDisplayMetrics();
+        initOCR();
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: This call uses `candyOrder` before it's initialized correctly. Fixed in c9521d0.

**Pull request merged**